### PR TITLE
feat: support to terra-classic v2.1.1

### PR DESCRIPTION
**Pull request by JoowonYun**

### CODE REVIEW

File: src/rest/useAutoRouter.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ const useAutoRouter = (params: Params) => {
 
       const tokenInfo = tokenInfos.get(to)
       const e = Math.pow(10, tokenInfo?.decimals || 6)
+      console.log(`11111`)
```
Comment (jhlee-young) [inline src/rest/useAutoRouter.ts:88]: could you delete this line?

Comment (JoowonYun) [reply src/rest/useAutoRouter.ts]: https://github.com/terraswap/classic-terraswap-web-app/pull/160/commits/5219cb1ebc0dad203f6a1de8b6b3515a5d9974cb

**Comment by jhlee-young** [comment]
`DenomBalanceResponse` also needs to be updated.
```
interface DenomBalanceResponse {
  balances: DenomInfo[]
}
```
, and please update url in useAPI.loadDenomBalance to `${fcd}/cosmos/bank/v1beta1/balances/${address}`

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by honeymaro — APPROVED** [review]

**Pull request merged**